evap/results/exporters.py

```diff
@@ -215,15 +233,18 @@ def write_overall_results(self, evaluations_with_results, course_results_exist):
 
         if course_results_exist:
             # Only query the number of evaluations once and keep track of it here.
-            count_gt_1 = [e.course_evaluations_count > 1 for e in evaluations]
+            count_gt_1 = [e.course_evaluations_count > 1 for e in evaluations]  # type: ignore[attr-defined]
```

Comment:
cast-to-any logic applies here (and in all other places that use `# type: ignore[attr-defined]`) as well.